gems/pending/spec/util/extensions/miq-module_spec.rb

```diff
@@ -18,7 +18,7 @@ module ::TestModule; end
     end
 
     after(:each) do
-      $miq_cache_with_timeout.clear
+      Module.clear_all_cache_with_timeout
```

Comment:
I'd rather keep the miq-module tests isolated...ultimately they should be ripped out into, probably, more_core_extensions, so it's good to know what is explicitly needed here.